### ISSUE TITLE
Migrating from 1.2 to latest version

### ISSUE THREAD

**Issue by tomeriko** [nuxsmin/sysPass#1446]
Hi there!

I am currently using sysPass version 1.2.0.11 on server version:  5.5.62-0+deb8u1
What is the best way to upgrade our server to the latest version?

Thanks,

Tomer

**Comment by nuxsmin** [comment]
Hi, please upgrade to the latest version of v1, then to v2 and finally to v3. Why such way?: because there has been a lot of database changes that cannot be carried out through every version.

**Comment by tomeriko** [comment]
Hi nuxsmin,

I am having trouble with the upgrade process.
I have downloaded the latest version (1.2.0.21) and created a new folder on /var/www/html/ that contains the new versoin folders and files.
I have set the required permissions and copied the config folder content of the old versions to the new one.
After that, I have browsed to the new app folder and I get the "Installation 1.2" page which requires users and passwords. I have tried to give the required accounts but I get the same error:
"Enter the admin's password
Application administrator's password"
